app/src/pages/inside/filtersPage/filterGrid/displayFilter/displayFilter.jsx

```diff
@@ -7,31 +7,37 @@ import styles from './displayFilter.scss';
 
 const cx = classNames.bind(styles);
 
-export const DisplayFilter = ({ userFilters, filter, onChangeDisplay }) => (
-  <Fragment>
-    <div className={cx('mobile-label', 'display-label')}>
-      <FormattedMessage id={'DisplayFilter.display'} defaultMessage={'Display on launches:'} />
-    </div>
-    <div className={cx('switcher-wrapper')}>
-      <InputSwitcher
-        value={userFilters.indexOf(filter.id) !== -1}
-        onChange={() => onChangeDisplay(filter.id)}
-      >
-        <span className={cx('switcher-label')}>
-          {userFilters.indexOf(filter.id) !== -1 ? (
-            <FormattedMessage id={'DisplayFilter.showOnLaunchesSwitcherOn'} defaultMessage={'ON'} />
-          ) : (
-            <FormattedMessage
-              id={'DisplayFilter.showOnLaunchesSwitcherOff'}
-              defaultMessage={'OFF'}
-            />
-          )}
-        </span>
-      </InputSwitcher>
-    </div>
-    <div className={cx('separator')} />
-  </Fragment>
-);
+export const DisplayFilter = ({ userFilters, filter, onChangeDisplay }) => {
+  const isFilterDisplayed = userFilters.find((item) => item.id === filter.id);
```

Comment:
`const isFilterDisplayed = !!userFilters.find((item) => item.id === filter.id);`
`is<something>` usually means that it's a boolean

Reply:
Sure. Fixed